dataland-community-manager/src/test/kotlin/org/dataland/datalandcommunitymanager/services/DataRequestAlterationManagerTest.kt

```diff
@@ -288,4 +316,48 @@ class DataRequestAlterationManagerTest {
         assertFalse(originalModificationTime == dummyDataRequestEntity.lastModifiedDate)
         assertEquals(RequestPriority.High, dummyDataRequestEntity.requestPriority)
     }
+
+    @Test
+    fun `validate that no email is send when the request status changes to Open`() {
+        // to properly test this we would need to set a request to non-sourceable
```

Comment:
remove comments/remove tests if this is part of an E2E test

Reply:
removed it